src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -0,0 +1,70 @@
+#ifndef RAY_GCS_NODE_MANAGER_H
+#define RAY_GCS_NODE_MANAGER_H
+
+#include <ray/common/id.h>
+#include <ray/protobuf/gcs.pb.h>
+#include <ray/rpc/client_call.h>
+
+namespace ray {
+
+namespace gcs {
+class RedisGcsClient;
+/// The GcsNodeManager is responsible for detecting the death of raylet, it has all
+/// functionality of raylet_monitor.
```

Comment:
I think we can remove "it has all functionality of raylet_monitor.", because in the future there is no raylet monitor. 
How about just `GcsNodeManager is responsible for managing and monitoring nodes.`